cmds/record.c

```diff
@@ -2023,6 +2024,13 @@ int do_child_exec(int ready, struct opts *opts,
 		char *s, *p;
 		int i;
 
+		/* current working directory can be changed during shell-script execution
+		 * so convert uftrace data directory path that can be relative
+		 * to an absolute path.
+		 */
```

Comment:
What about changing the comment as follows?
```c
		/*
                 * The current working directory can be changed by calling chdir.
		 * So dirname has to be converted to an absolute path to avoid unexpected problems.
		 */
```

Reply:
Looks better !